tests/basic/misc_test.go

```diff
@@ -1157,3 +1161,102 @@ var _ = Describe("{AutoFSTrimReplAddWithNoPool0}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+// Add a test to detach all the disks from a node and then reattach them to a different node
+var _ = Describe("{NodeDiskDetachAttach}", func() {
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("NodeDiskDetachAttach", "Validate disk detach and attach", nil, 0)
+	})
+	var contexts []*scheduler.Context
+
+	testName := "nodediskdetachattach"
+	stepLog := "has to detach all disks from a node and reattach them to a different node"
+	It(stepLog, func() {
```

Comment:
How about log.Infof after It to help while debugging?

Reply:
there is a log.InfoD